### ISSUE TITLE
Add Typical similar to a TV IR Remote

### ISSUE THREAD

**Issue by juanpintom** [souliss/soulissapp#138]
Hi, im working to control more devices via IR and will be good to have a Typical similar to a TV Remote like this:
![wpid-photostrip-2013-12-11-1](https://cloud.githubusercontent.com/assets/7034151/18707649/5a217398-7ff7-11e6-9943-f80b724a4162.jpg)

Can be a clone of T15 or T32.

Other good idea will be a "configurable with labels" T15, just the same of the T15 function but add a Rename to the label with a Long Press on the button.

Thanks

**Comment by jkutianski** [comment]
@juanpintom
I think is better if you send remote codes & code table id and not a button press signal.
On this case you can make a receiver & transmitter with the same Typical.

**Comment by jkutianski** [comment]
Related to https://github.com/souliss/souliss/issues/277

**Comment by jkutianski** [comment]
+1

**Comment by juanpintom** [comment]
Yes I agree, if we can set IR codes at runtime on the app will be more
powerful for sure!

Thanks

2016-10-04 3:54 GMT+02:00 Juan Pablo Kutianski notifications@github.com:
